src/containerapp/azext_containerapp/tests/latest/test_containerapp_patch.py

```diff
@@ -0,0 +1,159 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+from unittest import mock
+import time
+import requests
+import docker
+
+from azure.cli.testsdk.reverse_dependency import get_dummy_cli
+from azure.cli.testsdk.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, JMESPathCheck, live_only)
+from knack.util import CLIError
+
+from azext_containerapp.tests.latest.common import TEST_LOCATION
+from azext_containerapp import _utils
+from azext_containerapp.tests.latest.utils import create_and_verify_containerapp_up
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+class ContainerAppPatchTest(ScenarioTest):
+    @live_only()
```

Comment:
May I ask why do you need to mark these new tests as `@live_only()`?

Reply:
@zhoxing-ms, these tests are marked `@live_only()` because they require docker to be running. Since the CI does not have `docker` installed they would not work if they aren't run live.

Reply:
Got it, thanks~